src/plugins/discover/public/application/main/components/no_results/no_results_suggestions/use_fetch_occurances_range.ts

```diff
@@ -66,7 +66,10 @@ export const useFetchOccurrencesRange = (params: Params): Result => {
             abortSignal: abortControllerRef.current?.signal,
           });
         } catch (error) {
-          //
+          if (error.name !== 'AbortError') {
+            // eslint-disable-next-line no-console
+            console.error(error);
+          }
```

Comment:
What was the error there?

I think we better use Kibana logging or remove this statement.

Reply:
I'm not actually sure since it was previously swallowing the error, and it didn't fail again in the test runner after I made my test changes. This is here because I want to ensure that if it fails again in the future there are console logs in the functional test output to help with troubleshooting. I was considering throwing here instead which would also help troubleshoot future failures, but I was worried about impacting the UX if it occurs for a user in production.

I don't have any experience with Kibana logging and couldn't find any examples in the public code. Do you know how to use it and if it also outputs to the console to make sure we can troubleshoot any future test failures here?

Reply:
Yeah, I just checked and could only find it in server code. Okay, let's keep the console error 👍